docs/release/releasing.md

```diff
@@ -7,10 +7,10 @@ It can be used locally or in CI.
 
 ## GitHub Actions
 GitHub Actions use Gradle to release the library.
-A release build can be [triggered manually by "workflow dispatch"](trigger-gha-release.mp4).
+1. A release build can be [triggered manually by "workflow dispatch"](trigger-gha-release.mp4).
```

Comment:
It's hard to conclude from `releasing.md` whether this is a description of GitHub or instruction to follow. I want to make it more precise to avoid confusion.
What consumer should know from the headers like this?
* Gradle
* GitHub Actions
* Current version in source


🤔

Reply:
Headers might be confusing. But a split ordered list doesn't fix that.

Reply:
I've pushed new proposal;)

Reply:
Give me feedback if you have a bigger idea how it should look like.
I 'm open to edit it.